### PR TITLE
AGDIGGER-262 Android debug builds fail with build tools 26

### PR DESCRIPTION
**Pull request by vchepeli**

### CODE REVIEW

**Comment by vchepeli** [comment]
ping @omatskiv

**Comment by ghost** [comment]
ping @matskiv

File: roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2 [hunk 1]
```diff
@@ -16,7 +16,7 @@ data:
         <org.csanchez.jenkins.plugins.kubernetes.volumes.PersistentVolumeClaim>
           <mountPath>{{ android_sdk_home }}</mountPath>
           <claimName>{{ ag_android_sdk.pv_claim.name }}</claimName>
-          <readOnly>true</readOnly>
+          <readOnly>{{ ag_android_sdk.pv_claim.read_only }}</readOnly>
```
Comment (matskiv) [inline roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2:19]: Disabling read-only status of PV invalidates build isolation.
SDK pv is shared between builds. Once writes are enabled, you can inject malicious scripts from your builds or simply install or remove parts of SDK.

**Review by matskiv — CHANGES_REQUESTED** [review]
This doesn't fix AGDIGGER-262 issue, it's just a workaround that invalidates security measures and introduces a path for breaking the build system.

File: roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2 (hunk 1 above)
Comment (ghost) [reply roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2:19]: this was related to jenkins configuration mainly which is not saved after save/apply in jenkins ui. Dont think we want readOnly jenkins instance

Comment (matskiv) [reply roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2:19]: But this is not a template for Jenkins. This is a template for android slave configuration.

Comment (ghost) [reply roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2:19]: Thats bug! Jenkins configuration could not be saved, @odra WDYT?

**Pull request closed without merging**